charts/hlf-k8s/README.md

```diff
@@ -240,6 +241,21 @@ monitor:
 
 Check the pod's logs to get the list of organizations currently present in each channel.
 
+### Discover monitor pod
+
+The discover monitor pod periodically displays the list of organizations that are connected to the application channel with the hlf [service discovery](https://hyperledger-fabric.readthedocs.io/en/release-1.4/discovery-cli.html).
+
+It is a convenience feature which facilitates troubleshooting.
+
+To enable it, on the peer:
+
+```
+discover-monitor:
+   enabled: true
+```
+
+Check the pod's logs to get the list of organizations currently connected in the application channel.
```

Comment:
This is a duplicate of the first sentence

Reply:
I don't think so, I use the same pattern as previous :/